lib/block-supports/typography.php

```diff
@@ -438,11 +440,13 @@ function gutenberg_get_computed_fluid_typography_value( $args = array() ) {
  *     @type string           $slug Kebab-case unique identifier for the font size preset.
  *     @type string|int|float $size CSS font-size value, including units where applicable.
  * }
- * @param bool  $should_use_fluid_typography An override to switch fluid typography "on". Can be used for unit testing. Default is `false`.
+ * @param bool|array $settings Optional Theme JSON settings array that overrides any global theme settings.
+ *                             As a bool, it acts as an override to switch fluid typography "on" (`true`) or "off" (`false`).
+ *                             Can be used for unit testing. Default is `array()`.
  *
  * @return string|null Font-size value or `null` if a size is not passed in $preset.
  */
-function gutenberg_get_typography_font_size_value( $preset, $should_use_fluid_typography = false ) {
+function gutenberg_get_typography_font_size_value( $preset, $settings = array() ) {
```

Comment:
🤦🏻  

It should have been something like this in the first place.

I'm wondering if I shouldn't just change this to a general `$options` array to make adding further vars in the future easier?

So we'd be dealing with `$options['settings']`.

Reply:
Actually, `$settings` makes sense I think as it harmonizes the PHP and JS function signatures.

$settings is an array already and can be extended.